src/main/scala/com/fulcrumgenomics/util/DelimitedDataParser.scala

```diff
@@ -65,6 +65,12 @@ class Row private[util] (private val headerIndices: Map[String,Int], private val
 
   /** Gets a value from the specified column. If the value is null or empty returns None. */
   def get[A](column: String)(implicit tag: ru.TypeTag[A]): Option[A] = get(headerIndices(column))
+
+  /** Gets a value from the specified column. If the no column with the given name exists, or the value is null or empty
+    * returns None. */
+  def getOrNone[A](column: String)(implicit tag: ru.TypeTag[A]): Option[A] = if (headerIndices.keys.exists(_ == column)) {
```

Comment:
A few thoughts:
1) This needs a clearer name since `get()` already returns `Option`. How about instead of a separate method, having `allowMissingColumn: Boolean = false` on `get()`?
2) We could also just make this a parameter to delimited data parser, so it's a mode for the parser?
3) I think you could implement the current version as `headerIndices.get(column).map(get[A](_))` and use the fact that map.get returns an `Option`.